exaudfclient/base/javacontainer/test/cpp/javacontainer_ctpg_test.cc

```diff
@@ -107,3 +110,19 @@ TEST(JavaContainer, import_script_with_escaped_options) {
                                                             "-XX:+UseSerialGC" };
     EXPECT_EQ(vm.getJavaVMInternalStatus().m_jvmOptions, expectedJVMOptions);
 }
+
+TEST(JavaContainer, basic_jars_order_remains) {
```

Comment:
The test name doesn't fit its content

Reply:
It does implicitely: If the order would changed to alphabetical order, the exception would be different